src/package/packageVersionCreate.ts

```diff
@@ -303,6 +303,7 @@ export class PackageVersionCreate {
       SourceOrg: this.options.sourceorg,
       CalculateCodeCoverage: this.options.codecoverage || false,
       SkipValidation: this.options.skipvalidation || false,
+      Language: this.options.language,
```

Comment:
@sethmuss does including language in the request need to be guarded with api version?

Reply:
@peternhale Good question. I went with no because the app server gives a decent error: "No such column 'Language' on sobject of type Package2VersionCreateRequest" 

If you think it's better to have a more specific error I can add. Let me know what you think!

Reply:
(that error is only passed back if the user tries to send in a language via sfdx-project.json or cmd line param)

Reply:
@sethmuss On the command side, will the flag to specify language have a default value?

Reply:
@peternhale Yes. The app server defaults the language value to the auth'd dev hub user's language. For backward compatibility and a couple other reasons.